src/apps/commands/management/commands/generate_data.py

```diff
@@ -28,7 +32,20 @@ def handle(self, *args, **kwargs):
         size = kwargs.get('size') or 3
         no_admin = kwargs.get('no_admin')
         print(f'Creating data of size {size} {"without an admin account." if no_admin else "with an admin account." }')
-        users = [UserFactory(username='admin', super_user=True) if i == 0 and not no_admin else UserFactory() for i in range(size)]
+        users = []
+        for i in range(size):
+            if i == 0 and not no_admin:
+                try:
+                    user = UserFactory(username='admin', super_user=True)
+                    user.set_password('admin')
```

Comment:
you can pass password to the user factory so you don't need to call set password. and the standard password is test, do we need to change that? if so, could just do it on the factory's post generation hook

Reply:
✔️

Reply:
_(keeping password as admin//admin -- can do admin//test if we all prefer that?)_